### PR TITLE
pass pylint

### PR DESCRIPTION
**Pull request by Excalartur**

### CODE REVIEW

File: gunicorn/util.py [hunk 1]
```diff
@@ -547,12 +547,12 @@ def seed():
         random.seed('%s.%s' % (time.time(), os.getpid()))
 
 
-def check_is_writeable(path):
+def check_is_writable(path):
     try:
-        f = open(path, 'a')
+        with open(path, 'a', encoding="utf-8") as f:
```
Comment (benoitc) [inline gunicorn/util.py:552]: is encoding really needed?

Comment (benoitc) [reply gunicorn/util.py:552]: @Excalartur ?

Comment (Excalartur) [reply gunicorn/util.py:552]: The linter fix addresses the pylint warning W1514 (unspecified-encoding)
 as explained in PEP 597: https://peps.python.org/pep-0597/
If you believe it's not critical, we can add this warning to the ignore list. What do you think?

Comment (benoitc) [reply gunicorn/util.py:552]: @Excalartur Maybe a little too cautious but I'm not sure it own't introduce some unknown breakage. So let's ignore it for now. Can you make the change?

Comment (Excalartur) [reply gunicorn/util.py:555]: I suspect that I have messed up with the rebase 😅
Can this PR be saved or is it doomed?

Comment (benoitc) [reply gunicorn/util.py:555]: we can keep it there I think it will work (unsure :)

Comment (Excalartur) [reply gunicorn/util.py:555]: Created a new PR #2996  :)

**Pull request closed without merging**